modules/ting_fulltext/ting_fulltext.module

```diff
@@ -67,15 +75,17 @@ function ting_fulltext_object_load($object_id) {
 function ting_fulltext_preprocess_ting_relation(&$variables) {
   /** @var \TingRelation $relation */
   $relation = $variables['relation'];
-
   if (!$variables['online'] && (NULL !== $relation->getObject() && NULL != $relation->getObject()->getTingObject())) {
     /** @var \OpenSearch\OpenSearchTingObject $ting_object */
     $ting_object = $relation->getObject()->getTingObject();
 
     // This will only work for the opensearch search provider, so check what
     // type of object we have.
     if ($ting_object instanceof OpenSearchTingObject && in_array('docbook', $relation->getObject()->getTingObject()->getFormatsAvailable())) {
-      $variables['fulltext_link'] = l(t('See the full-text version'), 'ting/object/' . $relation->uri . '/fulltext');
+      // For performance reasons we load the review in full text as ajax.
+      drupal_add_js(drupal_get_path('module', 'ting_fulltext') . '/js/ting_fulltext.js');
+      $url = url('ting/object/' . $relation->uri . '/fulltext/ajax');
+      $variables['abstract'] = '<div id="js-ting-lektor-fulltext" data-fulltext-url="' . $url . '">' . $variables['abstract'] . '</div>';
```

Comment:
Please do not use HTML ids for elements that you cannot guarantee to be unique.

As far as I can tell it should be fine to use `class="js-ting-lektor-fulltext"` instead of `id="js-ting-lektor-fulltext"` and update the behavior accordingly.